docker-app/qfieldcloud/core/models.py

```diff
@@ -1810,15 +1810,27 @@ def save(self, recompute_storage=False, *args, **kwargs):
                 )["file_storage_bytes"]
 
         # Ensure that the Project's storage_keep_versions is at least 1, and reflects the plan's default storage_keep_versions value.
+        storage_keep_versions_modified = False
         if not self.storage_keep_versions:
             self.storage_keep_versions = (
                 self.owner.useraccount.current_subscription.plan.storage_keep_versions
             )
+            storage_keep_versions_modified = True
 
         assert self.storage_keep_versions >= 1, (
             "If 0, storage_keep_versions mean that all file versions are deleted!"
         )
 
+        if "update_fields" in kwargs and kwargs["update_fields"] is not None:
+            update_fields = set(kwargs["update_fields"])
+            if recompute_storage:
+                update_fields.add("file_storage_bytes")
+
+            if storage_keep_versions_modified:
```

Comment:
```suggestion
            if not self.storage_keep_versions:
```

Instead of renaming the same thing in a new variable, can't we use the original source for the boolean flag above?